source/docs/casper/concepts/design/rewards.md

```diff
@@ -78,20 +80,18 @@ round_seigniorage_rate = [7, 175070816]
 
 The following steps summarize the rewards distribution mechanism.
 
-Each round has a reward pool set in the chainspec under the `round_seigniorage_rate`.
+Each round has a reward pool calculated from the `round_seigniorage_rate` chainspec parameter and the current total supply for the relevant era.
 
-In each round, the reward pool is split into two parts for block creation and finality signatures. The split is configurable using the `finders_fee` setting in the chainspec:
-- A percentage is allocated for block creation.
-- A percentage is allocated for finality signatures.
+In each round, the reward pool is split into two parts for block proposals and finality signature rewards, based on the `finality_signature_proportion` chainspec parameter. 
 
-The amount allocated for finality signatures is split further into two parts: creating and collecting finality signatures. The split is configurable in the chainspec using the `finality_signature_proportion` setting. 
+The amount allocated for finality signatures is split further into two parts: creating and publishing finality signatures. The split is configurable in the chainspec using the `finders_fee` chainspec parameter. 
 
 For each finality signature:
-- The creator gets a portion of what was allocated for creating signatures, proportional to its weight.
-- The block proposer gets a portion of the collection allocation proportional to the creator's weight.
+- The creator gets a portion of what was allocated for creating signatures, in proportion to relative consensus weight.
+- The block proposer gets a portion of corresponding to the `finders_fee` chainspec parameter, scaled by total collected signature creator weight divided by total weight in the relevant era.
 
 <p align="center">
 <img src={"/image/design/rewards-pot.png"} alt="Pie chart showing how rewards are split" width="300"/>
 </p>
 
-The rewards calculation takes place at the end of an era. In addition to rewarding everything in that era, the algorithm also looks back into previous era(s), depending on the `signature_rewards_max_delay` parameter, to reward signatures on the switch blocks.
+The rewards calculation takes place at the end of an era. In addition to rewarding everything in that era, the algorithm also looks back into previous era(s), depending on the `signature_rewards_max_delay` parameter, to compensate for delay in creation and distribution of finality signature. In particular, this enables us to reward switch blocks, which was impossible with the prior, Highway-specific calculation.
```

Comment:
Maybe it should just say "era" pending clarification from Bart on whether we allow to look back more than one era. But it's correct in principle. We look back _into_ a previous era, but look back _at_ prior blocks.

Reply:
Q&A from Bart:
Is it possible to look back into multiple eras in the past?
Technically it would be possible, but it would require some eras to be shorter than the lookback window, which is something like 3 or 5 blocks, so it shouldn't happen under normal operation. (We have single-block eras after every upgrade, but we don't look back across an upgrade boundary, so this case is excluded.)

As a result, I've made this update: "the algorithm also looks back into blocks from the previous era".

I think we can go with this to close your PR and then send the main content to Bart for final review.